OpenSim/Common/Component.cpp

```diff
@@ -1791,11 +1791,14 @@ void Component::warnBeforePrint() const {
         }
     }
     if (!message.empty()) {
-        std::cout << "Warning in " << getConcreteClassName()
+        std::stringstream buffer;
+        buffer << "Warning in " << getConcreteClassName()
                 << "::print(): The following connections are not finalized "
                    "and will not appear in the resulting XML file. "
                    "Call finalizeConnections() before print().\n"
+                   "To ignore, please call Object::setDebugLevel(1) first.\n"
```

Comment:
Same comment as in `Object.h`. Could be something like `To ignore, please set _debugLevel to at least 1 (e.g, by calling Object::setDebugLevel(1))`

Reply:
I prefer not to show private variable names to API users in case we change how this is implemented in the future, also non C++ users have no choice but call setDebugLevel, so I'd prefer to keep user facing message at this level.

Reply:
Thanks for the clarification, this makes sense. I think it's fine as is, or a middle ground could be then just to have a similar message as the one above where it doesn't show the private variable name:
`To ignore, please set the debug level to at least 1 (e.g, by calling Object::setDebugLevel(1)) first.`